src/lu.jl

```diff
@@ -102,6 +102,39 @@ function lu!(A::HermOrSym{T}, pivot::Union{RowMaximum,NoPivot,RowNonZero} = lupi
     end
     lu!(A.data, pivot; check, allowsingular)
 end
+
+#reusing LU object
+#lu!(F::LU,A) should be dispatched on the type of matrix stored in the LU factorization.
+#but special care needs to be done in the HermOrSym case
+function _lu_copy!(A,x)
```

Comment:
you don't use an underscore in the other versions?

Also would like nicer if you called the second argument `X` or `B` to make it look more matrix-like